content/it/docs/reference/glossary/deployment.md

```diff
@@ -0,0 +1,20 @@
+---
+title: Deployment
+id: deployment
+date: 2018-04-12
+full_link: /docs/concepts/workloads/controllers/deployment/
+short_description: >
+  Gestisce una applicazione replicata nel tuo cluster.
+
+aka: 
+tags:
+- fundamental
+- core-object
+- workload
+---
+ Un oggetto API che gestisce un'applicazione replicatata, tipicamente esegue Pods senza stato locale.
+
+<!--more--> 
+Ogni replica è rappresentata da un {{< glossary_tooltip term_id="pod" >}}, e le Pods sono distribuite attraverso i 
```

Comment:
the term "Pod" has been considered male in previous translations. So: s/e le Pods sono deistribuite/i pod sono distribuiti/